plugins/source/azure/Makefile

```
@@ -23,21 +23,28 @@ install-hooks:
 gen-docs:
 	rm -rf ./docs/tables/*
 	go run main.go doc ./docs/tables
-	sed 's_(\(.*\))_(https://github.com/cloudquery/cloudquery/blob/main/plugins/source/azure/docs/tables/\1)_' docs/tables/README.md > ../../../website/pages/docs/plugins/sources/azure/tables.md
-	go run main.go doc --format json docs
-	go run scripts/policy_docs/main.go azure policies ../../../website/pages/docs/plugins/sources/azure/policies.md docs/__tables.json
-	rm docs/__tables.json
 
 .PHONY: lint
 lint:
 	golangci-lint run --config ../../.golangci.yml 
 
-.PHONY: gen-code
-gen-code:
-	grep -rl '// Code generated by codegen; DO NOT EDIT.' resources/services/* | xargs rm
-	go install golang.org/x/tools/cmd/goimports@latest
-	go run codegen/main.go
+.PHONY: codegen0
+codegen0:
+	grep -rl '// Code generated by codegen; DO NOT EDIT.' codegen1/recipes/* | xargs rm
+	go run codegen0/main.go
+
+.PHONY: codegen1
+codegen1:
+	grep -rl '// Code generated by codegen; DO NOT EDIT.' codegen2/recipes/* | xargs rm
+	go run codegen1/main.go
+
+.PHONY: codegen2
+codegen2:
+	grep -rl '// Code generated by codegen; DO NOT EDIT.' resources/* | xargs rm
+	grep -rl '// Code generated by codegen; DO NOT EDIT.' client/* | xargs rm
+	go run codegen2/main.go
+
+.PHONY: codegen
```

```suggestion
.PHONY: gen
```

This is our convention in all plugins